src/rfbzero/degradation.py

```diff
@@ -6,71 +6,111 @@
 
 
 class DegradationMechanism(ABC):
-    """Base class to be overridden by specific degradation mechanisms."""
+    """Abstract base class to be implemented by specific degradation mechanisms."""
 
     @abstractmethod
-    def degrade(self, c_ox: float, c_red: float, timestep: float) -> tuple[float, float]:
-        """Applies desired degradation mechanisms to oxidized/reduced species at each timestep."""
+    def degrade(self, c_ox: float, c_red: float, time_step: float) -> tuple[float, float]:
+        """Applies desired degradation mechanisms to oxidized/reduced species at each time step."""
         raise NotImplementedError
 
 
-class ChemicalDegradation(DegradationMechanism):
+class ChemicalDegradationOxidized(DegradationMechanism):
     """
-    Provides an N-th order chemical degradation mechanism.
+    Provides an N-th order chemical degradation mechanism for an oxidized species.
 
     Parameters
     ----------
     rate_order : int
         Rate order for chemical degradation reaction.
     rate_constant : float
         N-th order rate constant of chemical oxidation (unit is rate order-dependent).
```

Comment:
looks like I hadn't described this properly before, should say `...constant of chemical degradation of oxidized species`